api/alembic/versions/481905f1dfe8_merge_alembic_heads.py

```diff
@@ -0,0 +1,26 @@
+"""Merge heads
+
+Revision ID: 481905f1dfe8
+Revises: 8efe0e7b9712, 39806f02cdec
+Create Date: 2022-01-17 12:40:01.613726
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '481905f1dfe8'
+down_revision = ('8efe0e7b9712', '39806f02cdec')
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # No action necessary, just merging 2 Alembic version HEADs into one
```

Comment:
Migrations should be serial, no? What's the case for diverging migrations?

Reply:
It came about from that nightmare Alembic scenario I had, that I chatted with you & Sy about (where I foolishly tried to use the same .json source data for 2 different migrations).